src/app/services/mail/mail.service.ts

```diff
@@ -879,6 +879,22 @@ export class MailService {
       )
     )
   }
+
+  sendLocalDevMail = (
+    smsMessage: string,
+  ): ResultAsync<true, MailGenerationError | MailSendError> => {
+    const mailOptions: MailOptions = {
+      to: this.#officialMail,
+      from: this.#senderFromString,
+      html: smsMessage,
+      subject: '[mocktwilio] SMS OTP Verification',
+      replyTo: this.#officialMail,
+      bcc: this.#senderMail,
+    }
+    return this.#sendNodeMail(mailOptions, {
+      mailId: 'sendWarningMailForAdmin',
+    })
+  }
```

Comment:
Hmm, this function does something more specific than its name suggests. `subject` should probably be an input arg instead of hardcoded.

Reply:
Agreed. Added subject as required params.